### DISCUSSION TITLE
Sonarcloud指摘事項の確認

### DISCUSSION THREAD

**Discussion by REI-MORI** [sinProject-Inc/talk#1449] [Q&A]
@iamsin Sonarcloudより以下の指摘がありました。
「ソースコードにハードコーディングしているIPアドレスが安全であるかどうか確認してください」

この実装を行っている箇所はVitestのテストコードで、このIPアドレスには実際にアクセスしていないため問題ないとの認識です。Reviewで「safe」としてよろしいでしょうか。
https://sonarcloud.io/project/security_hotspots?id=sinProject-Inc_talk&pullRequest=1446&sinceLeakPeriod=true&tab=code

**Comment by sinproject-iwasaki** [comment]
@REI-MORI 

こちらは safe で大丈夫です！
